### PR TITLE
Add app server lines 376 to 381

### PR DESCRIPTION
**Pull request by timgerstel**
Signed-off-by: Timothy Gerstel <tgerstel@rocketsoftware.com>

### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../docs/contribute/contributing.md) to this repository.

- [ ] If the changes in this PR is part of the next future release, make this pull request against the **docs-staging** branch which will be published at the next release boundary. If the changes in this PR are part of the current release, use the default base branch, **master**. For more information about branches, see https://github.com/zowe/docs-site/tree/master#understanding-the-doc-branches. 
      
- [ ] If this PR relates to GitHub issues in `docs-site` or other repositories, please list in Description, prefixed with **close**, **fix** or **resolve** keywords.

### Description (including links to related git issues)
Please describe your pull request.

:heart:Thank you!

### CODE REVIEW

File: docs/troubleshoot/app-framework/appserver-error-codes.md [hunk 1]
```diff
@@ -3243,68 +3243,68 @@ The following error message codes may appear on the app-server log. Use the foll
...
+  Contact the author of the component, or a system administrator.
 
 
 
 ### ZWED0155E":"(%s) is not a supported endpoint for %s. Skipping (%s)... Supported: %s
 
   **Reason:**
 
-  TODO
+  (_url_) is not a supported endpoint for _componentName_. Skipping _pluginid_... Supported: _urls_
```
Comment (1000TurquoisePogs) [inline docs/troubleshoot/app-framework/appserver-error-codes.md:3270]: you're just putting the message description in the reason.
the message description of all the lines in this PR are still broken.

you have
```
### ZWED0155E":"(%s) is not a supported endpoint for %s. Skipping (%s)... Supported: %s
```

you should have

```
### ZWED0155E

    (_url_) is not a supported endpoint for _componentName_. Skipping _pluginid_... Supported: _urls_

   **Reason:**

    something insightful
```

and so on, for all the messages.

File: docs/troubleshoot/app-framework/appserver-error-codes.md [hunk 2]
```diff
@@ -3243,68 +3243,68 @@ The following error message codes may appear on the app-server log. Use the foll
 
   **Reason:**
 
-  TODO
+  (_operatingSystemName_) is not a supported platform for _componentName_. Skipping _pluginid_... Supported: _requiredOperatingSystem_
```
Comment (1000TurquoisePogs) [inline docs/troubleshoot/app-framework/appserver-error-codes.md:3246]: The reason for the error is the plugindefinition states what is valid, and there is a mismatch.

File: docs/troubleshoot/app-framework/appserver-error-codes.md [hunk 3]
```diff
@@ -3243,68 +3243,68 @@ The following error message codes may appear on the app-server log. Use the foll
...
 
   **Action:**
 
-  TODO
+  Contact the author of the component, or a system administrator.
```
Comment (1000TurquoisePogs) [inline docs/troubleshoot/app-framework/appserver-error-codes.md:3250]: Or check if you're running an outdated version of the plugin, or remove the plugin, or migrate zowe to the supported platform.

File: docs/troubleshoot/app-framework/appserver-error-codes.md [hunk 4]
```diff
@@ -3243,68 +3243,68 @@ The following error message codes may appear on the app-server log. Use the foll
...
```
Comment (1000TurquoisePogs) [inline docs/troubleshoot/app-framework/appserver-error-codes.md:3262]: Or check if you're running an outdated version of the plugin, or remove the plugin, or migrate zowe to the supported architecture.

File: docs/troubleshoot/app-framework/appserver-error-codes.md [hunk 5]
```diff
@@ -3243,68 +3243,68 @@ The following error message codes may appear on the app-server log. Use the foll
...
 
   **Reason:**
 
-  TODO
+  (_url_) is not a supported endpoint for _componentName_. Skipping _pluginid_... Supported: _urls_
 
   **Action:**
 
-  TODO
+  Refer to component API doc.
```
Comment (1000TurquoisePogs) [inline docs/troubleshoot/app-framework/appserver-error-codes.md:3274]: i dont think users will know what you mean by this.
users need to confirm they're on the right version of zowe and the right version of the plugin, and then refer to the plugin documentation or contact the plugin developer for more info.

File: docs/troubleshoot/app-framework/appserver-error-codes.md [hunk 6]
```diff
@@ -3243,68 +3243,68 @@ The following error message codes may appear on the app-server log. Use the foll
...
 
 
 
 ### ZWED0156E":"Could not register default plugins into app-server
 
   **Reason:**
 
-  TODO
+  Could not register default plugins into app-server
```
Comment (1000TurquoisePogs) [inline docs/troubleshoot/app-framework/appserver-error-codes.md:3282]: Why? is there an error people can expect to find above or below this message?

File: docs/troubleshoot/app-framework/appserver-error-codes.md [hunk 7]
```diff
@@ -3243,68 +3243,68 @@ The following error message codes may appear on the app-server log. Use the foll
...
 
   **Action:**
 
-  TODO
+  Verify integrity of required plugin files, or contact publisher.
```
Comment (1000TurquoisePogs) [inline docs/troubleshoot/app-framework/appserver-error-codes.md:3286]: how to verify?
perhaps `zwe support`. linking to that would be good.

File: docs/troubleshoot/app-framework/appserver-error-codes.md [hunk 8]
```diff
@@ -3243,68 +3243,68 @@ The following error message codes may appear on the app-server log. Use the foll
...
 
 
 
 ### ZWED0157E":"Could not register default plugin %s into app-server
 
   **Reason:**
 
-  TODO
+  Could not register default plugin _pluginid_ into app-server
```
Comment (1000TurquoisePogs) [inline docs/troubleshoot/app-framework/appserver-error-codes.md:3294]: Why? is there an error people can expect to find above or below this message?

File: docs/troubleshoot/app-framework/appserver-error-codes.md [hunk 9]
```diff
@@ -3243,68 +3243,68 @@ The following error message codes may appear on the app-server log. Use the foll
...
 
   **Action:**
 
-  TODO
+  Verify integrity of required plugin files, or contact publisher.
```
Comment (1000TurquoisePogs) [inline docs/troubleshoot/app-framework/appserver-error-codes.md:3298]: perhaps zwe support. linking to that would be good.

File: docs/troubleshoot/app-framework/appserver-error-codes.md [hunk 10]
```diff
@@ -3243,68 +3243,68 @@ The following error message codes may appear on the app-server log. Use the foll
...
 
   **Reason:**
 
-  TODO
+  Could not listen on address (_ipAddress_):(_port_). Insufficient permissions to perform port bind.
 
   **Action:**
 
-  TODO
+  Contact system administrator.
```
Comment (1000TurquoisePogs) [inline docs/troubleshoot/app-framework/appserver-error-codes.md:3310]: would we see an exception above this message? perhaps they need to give the admin that message.
perhaps the port isnt valid, the ip isnt valid, or both of them are correct but the zowe runtime user just lacks permission.

**Review by 1000TurquoisePogs — CHANGES_REQUESTED** [review]
Formatting is broken and messages could use more insights.

File: docs/troubleshoot/app-framework/appserver-error-codes.md [hunk 11]
```diff
@@ -3239,72 +3239,72 @@ The following error message codes may appear on the app-server log. Use the foll
 
 
 
-### ZWED0153E":"(%s) is not a supported platform for %s. Skipping (%s)... Supported: %s
+### ZWED0153E":"(_operatingSystemName_) is not a supported platform for _componentName_. Skipping (_pluginid_)... Supported: _requiredOperatingSystem_
```
Comment (DivergentEuropeans) [inline docs/troubleshoot/app-framework/appserver-error-codes.md:3242]: These error messages are formatted incorrectly relative to previous PR's. We have been doing

 ie.
 
image

**Review by DivergentEuropeans — CHANGES_REQUESTED** [review]

File: docs/troubleshoot/app-framework/appserver-error-codes.md [hunk 12]
```diff
@@ -3239,72 +3239,72 @@ The following error message codes may appear on the app-server log. Use the foll
...
+  The endpoint _url_ does not match any required endpoints of _componentName_. Supported endpoints may be viewd in the component's pluginDefinition.json.
 
   **Action:**
 
-  TODO
+ Refer to _componentName_ pluginDefinition.json for supported endpoints. Optionally, remove _url_ from the required endpoints in pluginDefinition.json. Lastly, contact the author of the component, or a system administrator.
 
 
 
 ### ZWED0156E":"Could not register default plugins into app-server
```
Comment (DivergentEuropeans) [inline docs/troubleshoot/app-framework/appserver-error-codes.md:3278]: I'm having a hard time finding where we use this error code, do we use it?

File: docs/troubleshoot/app-framework/appserver-error-codes.md [hunk 13]
```diff
@@ -3239,72 +3239,72 @@ The following error message codes may appear on the app-server log. Use the foll
...
 
   **Action:**
 
-  TODO
+  Verify integrity of Zowe installation, or contact system administrator. Please refer to https://docs.zowe.org/stable/appendix/zwe_server_command_reference/zwe/support/zwe-support for collecting Zowe runtime information.
 
 
 
-### ZWED0157E":"Could not register default plugin %s into app-server
+### ZWED0157E":"Could not register default plugin _pluginid_ into app-server
```
Comment (DivergentEuropeans) [inline docs/troubleshoot/app-framework/appserver-error-codes.md:3290]: And this one, do we actually use it?

File: docs/troubleshoot/app-framework/appserver-error-codes.md (hunk 11 above)
Comment (timgerstel) [reply docs/troubleshoot/app-framework/appserver-error-codes.md:3292]: My understanding was to just replace string substitutions with more clear variable names, I don't recall discussing the change you mentioned. The reason after this line also provides the user with a further explanation.

Also if you scroll up in the same file you can find plenty of other messages that use the same formatting as this PR. Personally I don't find the difference in formatting/fillibustering to provide any additional clarity to the user.

File: docs/troubleshoot/app-framework/appserver-error-codes.md (hunk 12 above)
Comment (timgerstel) [reply docs/troubleshoot/app-framework/appserver-error-codes.md:3328]: Yes, try refining your search within the core zowe components to search for the numeric part of the error code + the severity character and exclude your search to js, ts, jsx, and tsx files.

**Review by DivergentEuropeans — APPROVED** [review]

**Review by 1000TurquoisePogs — APPROVED** [review]

**Pull request merged**